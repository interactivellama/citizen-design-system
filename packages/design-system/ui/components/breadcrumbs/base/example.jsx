// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import BreadCrumbs from '../index.react';
import ButtonIcon from '../../button-icons/';
import {
  Trigger,
  Menu,
  MenuList,
  MenuItem
} from '../../menus/dropdown/example';
const { Crumb } = BreadCrumbs;

const moreIcon = (
  <ButtonIcon
    className="slds-button_icon-border-filled slds-button_icon-x-small"
    symbol="threedots"
    assistiveText="Show More"
    aria-haspopup="true"
    title="Show More"
  />
);

let BreadcrumbMenu = props => (
  <Trigger className="slds-is-open" triggerIcon={moreIcon}>
    <Menu className="slds-dropdown_left slds-dropdown_actions">
      <MenuList>
        <MenuItem tabIndex="0">Menu Item One</MenuItem>
        <MenuItem>Menu Item Two</MenuItem>
        <MenuItem>Menu Item Three</MenuItem>
      </MenuList>
    </Menu>
  </Trigger>
);

export const Context = props => (
  <div className="demo-only" style={{ height: '150px' }}>
    {props.children}
  </div>
);

export const BreadcrumbsBase = props => (
  <BreadCrumbs>
    <Crumb href="javascript:void(0);">Parent Entity</Crumb>
    <Crumb href="javascript:void(0);">Parent Record Name</Crumb>
  </BreadCrumbs>
);

/* for snapshot tests */
export const BreadcrumbsWithOverflow = props => (
  <BreadCrumbs listClassNames="slds-grid_vertical-align-center">
    <Crumb hasMenu>
      <BreadcrumbMenu />
    </Crumb>
    <Crumb href="javascript:void(0);">Parent Entity</Crumb>
    <Crumb href="javascript:void(0);">Parent Record Name</Crumb>
  </BreadCrumbs>
);

/// ///////////////////////////////////////////
// Export
/// ///////////////////////////////////////////

/* export for snapshot tests */
export default <BreadcrumbsBase />;

/* export for site examples */
export let states = [
  {
    id: 'overflow-breadcrumbs',
    label: 'Breadcrumbs with Overflow Menu',
    element: <BreadcrumbsWithOverflow />
  }
];
