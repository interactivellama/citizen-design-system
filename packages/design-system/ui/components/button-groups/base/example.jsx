// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import ButtonIcon from '../../button-icons/';
import {
  Menu,
  MenuList,
  MenuItem,
  Trigger
} from '../../menus/dropdown/example';
import classNames from 'classnames';

/// ////////////////////////////////////////
// Partial(s)
/// ////////////////////////////////////////

export let ButtonGroup = props => (
  <div
    className={classNames('slds-button-group', props.className)}
    role="group"
  >
    {props.children}
  </div>
);

export let ButtonGroupList = props => (
  <ul className={classNames('slds-button-group-list', props.className)}>
    {props.children}
  </ul>
);

/// ////////////////////////////////////////
// Export
/// ////////////////////////////////////////

export default (
  <ButtonGroup>
    <button className="slds-button slds-button_neutral">Refresh</button>
    <button className="slds-button slds-button_neutral">Edit</button>
    <button className="slds-button slds-button_neutral">Save</button>
  </ButtonGroup>
);

export let states = [
  {
    id: 'disabled',
    label: 'Disabled',
    element: (
      <ButtonGroup>
        <button className="slds-button slds-button_neutral">Refresh</button>
        <button className="slds-button slds-button_neutral" disabled>
          Edit
        </button>
        <button className="slds-button slds-button_neutral">Save</button>
      </ButtonGroup>
    )
  },
  {
    id: 'with-overflow',
    label: 'With Overflow Menu',
    element: (
      <ButtonGroup>
        <button className="slds-button slds-button_neutral">Refresh</button>
        <button className="slds-button slds-button_neutral">Edit</button>
        <button className="slds-button slds-button_neutral">Save</button>
        <Trigger className="slds-button_last" ariaExpanded="false" />
      </ButtonGroup>
    )
  },
  {
    id: 'with-overflow-open',
    label: 'With Overflow Menu - Open',
    element: (
      <div className="demo-only" style={{ height: '8.75rem' }}>
        <ButtonGroup>
          <button className="slds-button slds-button_neutral">Refresh</button>
          <button className="slds-button slds-button_neutral">Edit</button>
          <button className="slds-button slds-button_neutral">Save</button>
          <Trigger className="slds-button_last slds-is-open">
            <Menu className="slds-dropdown_right slds-dropdown_actions">
              <MenuList>
                <MenuItem tabIndex="0">Overflow Item One</MenuItem>
                <MenuItem>Overflow Item Two</MenuItem>
                <MenuItem>Overflow Item Three</MenuItem>
              </MenuList>
            </Menu>
          </Trigger>
        </ButtonGroup>
      </div>
    )
  },
  {
    id: 'with-overflow-disabled',
    label: 'With Overflow Menu - Disabled',
    element: (
      <ButtonGroup>
        <button className="slds-button slds-button_neutral">Refresh</button>
        <button className="slds-button slds-button_neutral">Edit</button>
        <button className="slds-button slds-button_neutral">Save</button>
        <div className="slds-dropdown-trigger slds-dropdown-trigger_click slds-button_last">
          <ButtonIcon
            aria-haspopup="true"
            assistiveText="More Actions"
            className="slds-button_icon-border-filled"
            disabled
            symbol="down"
            title="More Actions"
          />
        </div>
      </ButtonGroup>
    )
  }
];

export let examples = [
  {
    id: 'inverse',
    label: 'Inverse',
    element: (
      <div
        className="demo-only"
        style={{ padding: '0.5rem', background: '#16325c' }}
      >
        <ButtonGroup>
          <button className="slds-button slds-button_inverse">Refresh</button>
          <button className="slds-button slds-button_inverse">Edit</button>
          <button className="slds-button slds-button_inverse">Save</button>
        </ButtonGroup>
      </div>
    )
  },
  {
    id: 'inverse-disabled',
    label: 'Inverse - Disabled',
    element: (
      <div
        className="demo-only"
        style={{ padding: '0.5rem', background: '#16325c' }}
      >
        <ButtonGroup>
          <button className="slds-button slds-button_inverse">Refresh</button>
          <button className="slds-button slds-button_inverse" disabled>
            Edit
          </button>
          <button className="slds-button slds-button_inverse">Save</button>
        </ButtonGroup>
      </div>
    )
  },
  {
    id: 'inverse-overflow',
    label: 'Inverse - With Overflow Menu',
    element: (
      <div
        className="demo-only"
        style={{ padding: '0.5rem', background: '#16325c' }}
      >
        <ButtonGroup>
          <button className="slds-button slds-button_inverse">Refresh</button>
          <button className="slds-button slds-button_inverse">Edit</button>
          <button className="slds-button slds-button_inverse">Save</button>
          <div className="slds-dropdown-trigger slds-dropdown-trigger_click slds-button_last">
            <ButtonIcon
              className="slds-button_icon-border-inverse"
              assistiveText="More Actions"
              aria-haspopup="true"
              symbol="down"
              title="More Actions"
            />
          </div>
        </ButtonGroup>
      </div>
    )
  },
  {
    id: 'inverse-overflow-disabled',
    label: 'Inverse - With Overflow Menu - Disabled',
    element: (
      <div
        className="demo-only"
        style={{ padding: '0.5rem', background: '#16325c' }}
      >
        <ButtonGroup>
          <button className="slds-button slds-button_inverse">Refresh</button>
          <button className="slds-button slds-button_inverse">Edit</button>
          <button className="slds-button slds-button_inverse">Save</button>
          <div className="slds-dropdown-trigger slds-dropdown-trigger_click slds-button_last">
            <ButtonIcon
              className="slds-button_icon-border-inverse"
              assistiveText="More Actions"
              disabled
              aria-haspopup="true"
              symbol="down"
              title="More Actions"
            />
          </div>
        </ButtonGroup>
      </div>
    )
  },
  {
    id: 'button-group-icon',
    label: 'Button Icon Group',
    element: (
      <ButtonGroup>
        <ButtonIcon
          className="slds-button_icon-border-filled"
          symbol="chart"
          assistiveText="Charts"
          title="Charts"
          aria-pressed="false"
        />
        <ButtonIcon
          className="slds-button_icon-border-filled"
          symbol="filterList"
          assistiveText="Filter List"
          title="Filter List"
          aria-pressed="false"
        />
        <ButtonIcon
          className="slds-button_icon-more"
          hasDropdown
          assistiveText="More Actions"
          title="More Actions"
        />
      </ButtonGroup>
    )
  },
  {
    id: 'button-group-icon-disabled',
    label: 'Button Icon Group - Disabled',
    element: (
      <ButtonGroup>
        <ButtonIcon
          className="slds-button_icon-border-filled"
          symbol="chart"
          assistiveText="Charts"
          title="Charts"
          aria-pressed="false"
        />
        <ButtonIcon
          className="slds-button_icon-border-filled"
          disabled
          symbol="filterList"
          assistiveText="Filter List"
          title="Filter List"
          aria-pressed="false"
        />
        <ButtonIcon
          className="slds-button_icon-more"
          hasDropdown
          assistiveText="More Actions"
          title="More Actions"
        />
      </ButtonGroup>
    )
  },
  {
    id: 'button-group-icon-selected',
    label: 'Button Icon Group - Selected',
    element: (
      <ButtonGroup>
        <ButtonIcon
          className="slds-button_icon-border-filled slds-is-selected"
          symbol="chart"
          assistiveText="Charts"
          title="Charts"
          aria-pressed="true"
        />
        <ButtonIcon
          className="slds-button_icon-border-filled"
          symbol="filterList"
          assistiveText="Filter List"
          title="Filter List"
          aria-pressed="false"
        />
        <ButtonIcon
          className="slds-button_icon-more"
          hasDropdown
          assistiveText="More Actions"
          title="More Actions"
        />
      </ButtonGroup>
    )
  }
];
