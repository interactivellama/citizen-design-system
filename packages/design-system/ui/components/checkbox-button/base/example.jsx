// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { CheckboxButton, CheckboxAddButtonDeprecated } from '../';

class CheckboxButtonExample extends Component {
  constructor(props) {
    super(props);
    this.state = {
      checked: !!this.props.isChecked,
      focused: false
    };
    this.handleCheckboxChange = this.handleCheckboxChange.bind(this);
    this.handleFocus = this.handleFocus.bind(this);
    this.handleBlur = this.handleBlur.bind(this);
  }

  handleCheckboxChange() {
    this.setState({
      checked: !this.state.checked
    });
  }

  handleFocus() {
    if (!this.state.focused) {
      this.setState({
        focused: true
      });
    }
  }

  handleBlur() {
    if (this.state.focused) {
      this.setState({
        focused: false
      });
    }
  }

  render() {
    const { iconChecked, iconUnchecked, isDisabled } = this.props;
    return (
      <CheckboxButton
        onChange={this.handleCheckboxChange}
        iconSymbol={this.state.checked ? iconChecked : iconUnchecked}
        isChecked={this.state.checked}
        isDisabled={isDisabled}
        isFocused={this.state.focused}
        onFocus={this.handleFocus}
        onBlur={this.handleBlur}
      />
    );
  }
}

CheckboxButtonExample.defaultProps = {
  iconChecked: 'check',
  iconUnchecked: 'add'
};

CheckboxButtonExample.propTypes = {
  iconChecked: PropTypes.string,
  iconUnchecked: PropTypes.string
};

export default <CheckboxButtonExample />;

export let states = [
  {
    id: 'checkbox-button-checked',
    label: 'Checked',
    element: <CheckboxButtonExample isChecked />
  },
  {
    id: 'checkbox-button-disabled',
    label: 'Disabled',
    element: <CheckboxButtonExample isDisabled />
  },
  {
    id: 'checkbox-button-checked-disabled',
    label: 'Checked and Disabled',
    element: <CheckboxButtonExample isChecked isDisabled />
  },
  {
    id: 'checkbox-button-icon-symbol',
    label: 'Different Icon',
    element: (
      <CheckboxButtonExample
        iconUnchecked="recycle_bin_empty"
        iconChecked="recycle_bin_full"
      />
    )
  },
  {
    id: 'checkbox-button-checked-icon-symbol',
    label: 'Different Icon - Checked',
    element: (
      <CheckboxButtonExample
        iconUnchecked="recycle_bin_empty"
        iconChecked="recycle_bin_full"
        isChecked
      />
    )
  },
  {
    id: 'checkbox-button-checked-disabled-icon-symbol',
    label: 'Different Icon - Checked and Disabled',
    element: (
      <CheckboxButtonExample
        iconChecked="recycle_bin_full"
        isChecked
        isDisabled
      />
    )
  },
  {
    id: 'checkbox-add-button-checked',
    label: 'Deprecated - Checked',
    element: <CheckboxAddButtonDeprecated checked />
  },
  {
    id: 'checkbox-add-button-disabled',
    label: 'Deprecated - Disabled',
    element: <CheckboxAddButtonDeprecated disabled />
  },
  {
    id: 'checkbox-add-button-checked-and-disabled',
    label: 'Deprecated - Checked and Disabled',
    element: <CheckboxAddButtonDeprecated checked disabled />
  }
];
