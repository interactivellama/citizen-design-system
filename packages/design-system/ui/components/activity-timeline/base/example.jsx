// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import { ButtonIcon } from 'ui/components/button-icons/base/example';
import { ActivityTimelineItem as Event } from 'ui/components/activity-timeline/event/example';
import { ActivityTimelineItem as Call } from 'ui/components/activity-timeline/call/example';
import { ActivityTimelineItem as Email } from 'ui/components/activity-timeline/email/example';
import { ActivityTimelineItem as Task } from 'ui/components/activity-timeline/task/example';
import SvgIcon from 'app_modules/ui/svg-icon';
import classNames from 'classnames';

export let TimelineIcon = props =>
  <div
    className={classNames(
      'slds-icon_container',
      props.symbol == 'log_a_call' ?
        'slds-icon-standard-log-a-call'
      : 'slds-icon-standard-' + props.symbol
    )}
    title={props.symbol == 'log_a_call' ? 'call' : props.symbol }
  >
    <SvgIcon
      className="slds-icon slds-icon--small"
      sprite="standard"
      symbol={ props.symbol }
    />
  </div>;

export let TimelineActions = props =>
  <div className="slds-timeline__actions">
    <p className="slds-timeline__date">{ props.date || 'Feb 24' }</p>
    <ButtonIcon
      className="slds-button--icon-border-filled slds-button--icon-x-small"
      symbol="down"
      assistiveText={ props.assistiveText || 'More Options' }
      aria-haspopup="true"
      title={ props.title || 'More Options' }
    />
  </div>;

export default (
  <ul className="slds-timeline">
    <li>
      <Task />
    </li>
    <li>
      <Event />
    </li>
    <li>
      <Call />
    </li>
    <li>
      <Email />
    </li>
  </ul>
);
