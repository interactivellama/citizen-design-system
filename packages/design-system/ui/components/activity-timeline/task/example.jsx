// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import MediaObject from 'ui/utilities/media-objects/index.react';
import { Checkbox } from 'ui/components/checkbox/base/example';
import { TimelineIcon, TimelineActions } from 'ui/components/activity-timeline/base/example';

export let ActivityTimelineItem = props =>
  <div className="slds-timeline__item">
    <span className="slds-assistive-text">Task</span>
    <MediaObject
      figureRight={
        <TimelineActions
          title="More Options for Task, Review proposals"
          assistiveText="More Options for Task, Review proposals"
        />
      }
    >
      <MediaObject
        className="slds-timeline__media slds-timeline__media--task"
        figureLeft={ <TimelineIcon symbol="task" /> }
        figureLeftClassName="slds-timeline__icon"
      >
        <MediaObject
          title="Review proposals for EBC deck with larger team and have marketing review this"
          figureLeft={ <Checkbox hideLabel label="mark-complete" /> }
        >
          <h3 className="slds-truncate" title="Review proposals for EBC deck with larger team and have marketing review this">
            <a href="javascript:void(0);">Review proposals for EBC deck with larger team and have marketing review this</a>
          </h3>
          <ul className="slds-list--horizontal slds-wrap">
            <li className="slds-m-right--large">
              <span className="slds-text-title">Contact:</span>
              <span className="slds-text-body--small"><a href="javascript:void(0);">Lei Chan</a></span>
            </li>
            <li className="slds-m-right--large">
              <span className="slds-text-title">Assigned to:</span>
              <span className="slds-text-body--small"><a href="javascript:void(0);">Betty Mason</a></span>
            </li>
          </ul>
        </MediaObject>
      </MediaObject>
    </MediaObject>
  </div>;

export default (
  <ActivityTimelineItem />
);
