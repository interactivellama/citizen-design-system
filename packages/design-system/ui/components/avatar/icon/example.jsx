// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import SvgIcon from 'app_modules/ui/svg-icon';
import { StandardIcon } from 'ui/components/icons/standard/example';
import { Avatar } from 'ui/components/avatar/base/example';
import classNames from 'classnames';


//////////////////////////////////////////////
// Export
//////////////////////////////////////////////

export default (
  <Avatar>
    <StandardIcon />
  </Avatar>
);

export let states = [
  {
    id: 'avatar-square-icon',
    label: 'Square Icon Fallback',
    element:
      <Avatar className="slds-avatar--medium">
        <StandardIcon />
      </Avatar>
  },
  {
    id: 'avatar-circle-icon',
    label: 'Circle Icon Fallback',
    element:
      <Avatar className="slds-avatar--circle slds-avatar--medium">
        <StandardIcon symbol="user" />
      </Avatar>
  },
  {
    id: 'size-x-small',
    label: 'Size - X Small',
    element:
      <Avatar className="slds-avatar--x-small">
        <StandardIcon />
      </Avatar>
  },
  {
    id: 'size-small',
    label: 'Size - Small',
    element:
      <Avatar className="slds-avatar--small">
        <StandardIcon />
      </Avatar>
  },
  {
    id: 'size-medium',
    label: 'Size - Medium',
    element:
      <Avatar className="slds-avatar--medium">
        <StandardIcon />
      </Avatar>
  },
  {
    id: 'size-large',
    label: 'Size - Large',
    element:
      <Avatar className="slds-avatar--large">
        <StandardIcon />
      </Avatar>
  }
];
