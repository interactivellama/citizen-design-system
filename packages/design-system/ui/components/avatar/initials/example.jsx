// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import { Avatar } from '../base/example';
import { AvatarInitials } from '../';

export default (
  <Avatar>
    <AvatarInitials
      iconClassName="slds-icon-standard-account"
      title="company name"
      initials="Ac"
    />
  </Avatar>
);

export let examples = [
  {
    id: 'linked-avatar-initials',
    label: 'Wrapped in a link',
    element: (
      <a href="javascript:void(0);">
        <Avatar>
          <AvatarInitials
            iconClassName="slds-icon-standard-account"
            title="company name"
            initials="Ac"
          />
        </Avatar>
      </a>
    )
  }
];
