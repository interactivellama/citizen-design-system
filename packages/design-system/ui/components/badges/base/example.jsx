// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import { UtilityIcon } from '../../icons/base/example';
import { Badge, InverseBadge, LightestBadge } from '../';

export default <Badge>Badge Label</Badge>;

export let examples = [
  {
    id: 'inverse',
    label: 'Inverse',
    element: <InverseBadge>Badge Label</InverseBadge>
  },
  {
    id: 'light-with-icon',
    label: 'Light with Icon',
    element: (
      <LightestBadge>
        <UtilityIcon
          containerClassName="slds-m-right_xx-small"
          className="slds-icon_xx-small slds-icon-text-default"
          assistiveText={false}
          symbol="moneybag"
        />
        423 Credits Available
      </LightestBadge>
    )
  },
  {
    id: 'badge-with-child',
    label: 'With Child Element',
    element: (
      <Badge>
        Component In: <strong>Lightning</strong>
      </Badge>
    )
  }
];
