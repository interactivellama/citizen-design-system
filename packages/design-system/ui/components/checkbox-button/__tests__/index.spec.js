/* eslint-env jest */
import React from 'react';
import { CheckboxAddButton } from '../base/example';

import createHelpers from '../../../../jest.helpers';

const { matchesMarkup } = createHelpers(__dirname);

it('renders a default checkbox button', () =>
  matchesMarkup(<CheckboxAddButton />));

it('renders a checked checkbox button', () =>
  matchesMarkup(<CheckboxAddButton checked />));

it('renders a disabled checkbox button', () =>
  matchesMarkup(<CheckboxAddButton disabled />));

it('renders a checked and disabled checkbox button', () =>
  matchesMarkup(<CheckboxAddButton checked disabled />));
