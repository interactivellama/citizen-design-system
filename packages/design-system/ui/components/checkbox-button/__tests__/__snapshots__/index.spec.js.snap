// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a checked and disabled checkbox button 1`] = `
<div
  className="slds-checkbox_add-button"
>
  <input
    className="slds-assistive-text"
    defaultChecked={true}
    disabled={true}
    id="add-checkbox-4"
    tabIndex={undefined}
    type="checkbox"
    value="add-checkbox-4"
  />
  <label
    className="slds-checkbox_faux"
    htmlFor="add-checkbox-4"
  >
    <span
      className="slds-assistive-text"
    >
      Add product
    </span>
  </label>
</div>
`;

exports[`renders a checked checkbox button 1`] = `
<div
  className="slds-checkbox_add-button"
>
  <input
    className="slds-assistive-text"
    defaultChecked={true}
    disabled={undefined}
    id="add-checkbox-2"
    tabIndex={undefined}
    type="checkbox"
    value="add-checkbox-2"
  />
  <label
    className="slds-checkbox_faux"
    htmlFor="add-checkbox-2"
  >
    <span
      className="slds-assistive-text"
    >
      Add product
    </span>
  </label>
</div>
`;

exports[`renders a default checkbox button 1`] = `
<div
  className="slds-checkbox_add-button"
>
  <input
    className="slds-assistive-text"
    defaultChecked={undefined}
    disabled={undefined}
    id="add-checkbox-1"
    tabIndex={undefined}
    type="checkbox"
    value="add-checkbox-1"
  />
  <label
    className="slds-checkbox_faux"
    htmlFor="add-checkbox-1"
  >
    <span
      className="slds-assistive-text"
    >
      Add product
    </span>
  </label>
</div>
`;

exports[`renders a disabled checkbox button 1`] = `
<div
  className="slds-checkbox_add-button"
>
  <input
    className="slds-assistive-text"
    defaultChecked={undefined}
    disabled={true}
    id="add-checkbox-3"
    tabIndex={undefined}
    type="checkbox"
    value="add-checkbox-3"
  />
  <label
    className="slds-checkbox_faux"
    htmlFor="add-checkbox-3"
  >
    <span
      className="slds-assistive-text"
    >
      Add product
    </span>
  </label>
</div>
`;
